server/src-lib/Hasura/Server/App.hs

```diff
@@ -232,6 +232,12 @@ class Monad m => MonadConfigApiHandler m where
 -- instance (MonadIO m, UserAuthentication m, HttpLog m, Tracing.HasReporter m) => MonadConfigApiHandler (Tracing.TraceT m) where
```

Comment:
Also this is unrelated to your PR. But I think let's remove this commented out code.